src/Components/BookDetails/BookDetails.tsx

```diff
@@ -1,39 +1,96 @@
 import React, { useEffect, useState } from 'react'
-import { useQuery, gql } from '@apollo/client'
+import { useQuery, gql, useMutation } from '@apollo/client'
 import { useParams } from 'react-router-dom'
 import './BookDetails.css'
 
 const BOOK_DETAILS = gql `
   query BookDetails($id: ID!) {
     book(id: $id) {
+      id
       title,
       summary,
       author,
       pageCount,
       imageUrl,
       users {
+        id
         userName,
         emailAddress,
         location
       }
     }
   }
 `
+const BORROW_BOOK = gql `
+  mutation BorrowBook( $userId: Int!, $bookId: Int!, $borrowerId: Int!) {
+    patchUserBook(input: {
+      userId: $userId
+      bookId: $bookId
+      borrowerId: $borrowerId
+      status: 1
```

Comment:
Great idea to start by hard coding this in until we figure out how to add it dynamically